### PR TITLE
Implement new stats/home page

### LINKED ISSUE #77: Main page improvements
- [x] On the main page, show all information that the API endpoint returns regarding the blockchain: https://developer.vocdoni.io/vocdoni-api/vochain-information
- [x] On the main page show also the transaction costs from the [API endpoint](https://developer.vocdoni.io/vocdoni-api/transaction-costs). In addition to the base cost and network capacity from [here](https://developer.vocdoni.io/vocdoni-api/price-factors-information)

### PR DESCRIPTION
**Pull request by selankon**
Fixes https://github.com/vocdoni/explorer/issues/77

Needs sdk: https://github.com/vocdoni/vocdoni-sdk/pull/406
Extended sdk: https://github.com/vocdoni/ui-components/pull/195

### CODE REVIEW

**Review by elboletaire — CHANGES_REQUESTED** [review]
Without having checked the code much, some friendly reminders for when we're back of holidays:

- The tokens component takes too long to load, and there are no pending calls so it's probably something in the implementation
- We cannot publish that right column this empty... we should get the stats back to the right side, and maybe even put the tokens info inside the blockchain card
- @jpaulet said the `i` modal is not very intuitive, maybe he wants to propose something for that

**Comment by selankon** [comment]
> Without having checked the code much, some friendly reminders for when we're back of holidays:
> 
>     * The tokens component takes too long to load, and there are no pending calls so it's probably something in the implementation
> 
>     * We cannot publish that right column this empty... we should get the stats back to the right side, and maybe even put the tokens info inside the blockchain card
> 
>     * @jpaulet said the `i` modal is not very intuitive, maybe he wants to propose something for that

It seems that it was an error during the build so you was looking an outdated version of the stats page. 

Check it now because the tokens component have a better behavior now.

File: src/components/Stats/ChainDetails/TxCosts.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    //   icon={Icons.TxIcon}
+    //   title={t('stats.voc_tokens', { defaultValue: 'VOC Tokens' })}
+    //   raw={txCosts}
+    //   isLoading={isLoading}
+    //   rightComp={<PriceFactorsModal />}
+    //   isError={isError}
+    //   error={error}
+    // >
+    <DetailsGrid rowGap={0} columnGap={6} details={prices} />
+    // </StatsCardWrapper>
```
Comment (elboletaire) [inline src/components/Stats/ChainDetails/TxCosts.tsx]: Remove commented code please

File: src/components/Stats/ChainDetails/StatsCards.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+interface IncrementalStatProps {
+  value: number
+  label: string
+}
+
+const IncrementalStat = ({ value, label }: IncrementalStatProps) => {
+  const [displayNumber, setDisplayNumber] = useState(0)
+  const duration = 2000
+
+  useEffect(() => {
```
Comment (elboletaire) [inline src/components/Stats/ChainDetails/StatsCards.tsx:46]: Comment please

**Review by elboletaire — CHANGES_REQUESTED** [review]
Two minor issues, other than that LGTM

File: src/components/Stats/ChainDetails/TxCosts.tsx (hunk 1 above)
Comment (selankon) [reply src/components/Stats/ChainDetails/TxCosts.tsx]: d2148c0

File: src/components/Stats/ChainDetails/StatsCards.tsx (hunk 2 above)
Comment (selankon) [reply src/components/Stats/ChainDetails/StatsCards.tsx:46]: d2148c0

**Pull request merged**